comprehensive-getting-started/Layer_Comprehensive_Getting_Started.ipynb

```diff
@@ -0,0 +1,868 @@
+        "\n",
+        "For more information about this function:<br>\n",
+        "https://docs.app.layer.ai/docs/sdk-library/layer-init\n",
+        "\n",
+        "---"
+      ],
+      "metadata": {
+        "id": "AQT4pThP1j5U"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# Step II: Project initialization and import some Layer packages\n",
+        "\n",
+        "# Matches and Sync with the project we created before and installs the required python packages on the Layer Remote\n",
+        "layer.init(\"my-first-project\",pip_packages=['matplotlib','sklearn'])\n",
+        "# Import functions from Layer\n",
+        "from layer.decorators import dataset, model, resources, fabric\n",
+        "from layer.decorators.assertions import assert_not_null, assert_true\n",
+        "from layer import Dataset,Model"
+      ],
+      "metadata": {
+        "id": "l6G61hqJlf19"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# **Step III:** Learning more about Dataset Versioning on Layer\n",
+        "---\n",
+        "Every time you run your data build function, Layer will automatically generate a new version of your dataset depending on changes in your function.<br><br>\n",
+        "This is the way how Layer does dataset versioning:<br><br>\n",
+        "**- If schema of a dataset is changed, then bump up the major version number**<br>\n",
+        "*(e.g v1.3 → v2.1)* <br>\n",
+        "**- Otherwise, bump up the minor version number <br>**\n",
+        "*(e.g v1.3 → v1.4)*"
+      ],
+      "metadata": {
+        "id": "LfBOajgPMZ0H"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "\n",
+        "\n",
+        "### Layer Decorators and Functions Used in Step-III\n",
+        "\n",
+        "---\n",
+        "\n",
+        "\n",
+        "\n",
+        "\n",
+        "**@dataset:**<br> \n",
+        "It is a decorator which defines the pandas data frame ***training_df*** returned by the function as a Layer dataset named ***my_first_dataset*** under the project ***my_first_project***. Layer starts tracking this dataset automatically every time you run this function. In other words, Layer will do versioning on your dataset and log any other data you want Layer to store along with this dataset. It is important to note that it is required that you could use this dataset decorator at the top a Python function returning a pandas data frame.<br><br>\n",
```

Comment:
`It is important to note that it is required that you could use this dataset decorator at the top a Python function returning a Pandas data frame.` Is `could` necessary here?

Reply:
removed